# PointAndMechaStateArray.msg

PointAndMechaState[] points  # パスポイントとMechaStateコマンドの配列